app/views/user/_exercises_table.erb

```diff
@@ -18,7 +18,7 @@
       <th>Language</th>
       <th>Exercise</th>
       <th>Iterations</th>
-      <th data-sorted="true" data-sorted-direction="descending">Latest Iteration</th>
+      <th data-sorted="true" data-sorted-direction="ascending">Latest Iteration</th>
```

Comment:
I believe "descending" is what it should be.

This property doesn't do anything to control the initial sort order of the list, it just reports what it is. (Or should be, I believe the initial data in the table is not currently sorted correctly.)